packages/next/cli/next-info.ts

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env node
+import os from 'os'
+import path from 'path'
+import childProcess from 'child_process'
+import fs from 'fs'
+
+import chalk from 'next/dist/compiled/chalk'
+import arg from 'next/dist/compiled/arg/index.js'
+import { printAndExit } from '../server/lib/utils'
+import { cliCommand } from '../bin/next'
+import isError from '../lib/is-error'
+
+const nextInfo: cliCommand = async (argv) => {
+  const validArgs: arg.Spec = {
+    // Types
+    '--help': Boolean,
+    // Aliases
+    '-h': '--help',
+  }
+  let args: arg.Result<arg.Spec>
+  try {
+    args = arg(validArgs, { argv })
+  } catch (error) {
+    if (isError(error) && error.code === 'ARG_UNKNOWN_OPTION') {
+      return printAndExit(error.message, 1)
+    }
+    throw error
+  }
+
+  if (args['--help']) {
+    console.log(
+      `
+      Description
+        Provides debug information about Next.js.
+        When opening a bug report, please include the result in the issue.
+
+      Usage
+        $ next info
+
+      Learn more: ${chalk.cyan(
+        'https://nextjs.org/docs/api-reference/cli#telemetry'
+      )}
+    `
+    )
+    return
+  }
+
+  console.log(`
+    Operating System:
+      Platform: ${os.platform()}
+      Arch: ${os.arch()}
+      Version: ${os.version()}
+    Binaries:
+      Node: ${process.version}
```

Comment:
```suggestion
      Node: ${process.versions.node}
```

I just realize we want the one without the `v` prefix